src/Application/ConfigResolver.php

```diff
@@ -76,4 +79,33 @@ public static function guess(string $directory): string
 
         return $preset;
     }
+
+    /**
+     * See https://www.php.net/manual/en/function.array-merge-recursive.php#96201
+     *
+     * @param mixed[] $base
```

Comment:
use array notation for arrays :)

Reply:
I already try `array<mixed>` but phpstan don't understand.
 
As it's a recursive function it's quite hard to find the correct notation :slightly_frowning_face: 

Anyway I can use it for the return